### PR TITLE
Add CentOS Stream to RHEL docs

### LINKED ISSUE #28438: Dotnet-SDK support for Centos Stream 8, 9 ?
I would be happy if you provide information about dotnet-sdk support for distro `Centos Stream 8` and `Centos Stream 9` which is the new red hat distro model (note: rolling release).

annoucement: https://www.redhat.com/en/topics/linux/what-is-centos-stream
and docker images provided: https://quay.io/repository/centos/centos?tab=tags

Quick test as of March 1st 2022
```sh
# docker run --rm -it quay.io/centos/centos:stream8
...
[root@43e8da7f83a7 /]# dnf search dotnet-sdk
Last metadata expiration check: 0:02:29 ago on Tue Mar  1 16:44:19 2022.
========================================================================================================== Name Matched: dotnet-sdk ===========================================================================================================
dotnet-sdk-2.1.x86_64 : .NET Core 2.1 Software Development Kit
dotnet-sdk-2.1.5xx.x86_64 : .NET Core 2.1.5xx Software Development Kit
dotnet-sdk-3.0.x86_64 : .NET Core 3.0 Software Development Kit
dotnet-sdk-3.1.x86_64 : .NET Core 3.1 Software Development Kit
dotnet-sdk-5.0.x86_64 : .NET 5.0 Software Development Kit
dotnet-sdk-6.0.x86_64 : .NET 6.0 Software Development Kit
```

and
```sh
# docker run --rm -it quay.io/centos/centos:stream9
...
[root@43e8da7f83a7 /]# dnf search dotnet-sdk
Last metadata expiration check: 0:00:07 ago on Tue Mar  1 16:47:37 2022.
========================================================================================================== Name Matched: dotnet-sdk ===========================================================================================================
dotnet-sdk-3.1.x86_64 : .NET Core 3.1 Software Development Kit
dotnet-sdk-6.0.x86_64 : .NET 6.0 Software Development Kit
```

note: we need it to support google/or-tools C++/.Net wrapper package on these distros... 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 577ce9b2-5b48-4ed2-c309-a78880015dd2
* Version Independent ID: 26300783-8df5-56a1-4ab6-4b8511e37f51
* Content: [Install .NET on CentOS - .NET](https://docs.microsoft.com/en-us/dotnet/core/install/linux-centos)
* Content Source: [docs/core/install/linux-centos.md](https://github.com/dotnet/docs/blob/main/docs/core/install/linux-centos.md)
* Product: **dotnet**
* Technology: **dotnet-install**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

- Added CentOS Stream to docs
- Update dates
- Validated supported Linux versions

Fixes #28438

@BillWagner for review

@omajid I linked to your blog post about installing .NET 3.1/5 on CentOS Stream 9.

### CODE REVIEW

File: docs/core/install/linux-rhel.md [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
+# Install the .NET SDK or the .NET Runtime on RHEL and CentOS Stream
 
 .NET is supported on Red Hat Enterprise Linux (RHEL). This article describes how to install .NET on RHEL.
 
 [!INCLUDE [linux-intro-sdk-vs-runtime](includes/linux-intro-sdk-vs-runtime.md)]
 
 ## Register your Red Hat subscription
 
-To install .NET from Red Hat on RHEL, you first need to register using the Red Hat Subscription Manager. If this hasn't been done on your system, or if you're unsure, see the [Red Hat Product Documentation for .NET](https://access.redhat.com/documentation/en-us/net/5.0).
+To install .NET from Red Hat on RHEL, you first need to register using the Red Hat Subscription Manager. If this hasn't been done on your system, or if you're unsure, see the [Red Hat Product Documentation for .NET](https://access.redhat.com/documentation/en-us/net/6.0).
```
Comment (omajid) [inline docs/core/install/linux-rhel.md:17]: Is it worth calling out that this step is not needed on CentOS Stream? Or is that going too much into the particulars of that distro?

File: docs/core/install/linux-rhel.md [hunk 2]
```diff
@@ -1,33 +1,35 @@
...
 author: adegeo
 ms.author: adegeo
-ms.date: 11/05/2021
+ms.date: 03/21/2022
 ---
 
-# Install the .NET SDK or the .NET Runtime on RHEL
+# Install the .NET SDK or the .NET Runtime on RHEL and CentOS Stream
 
 .NET is supported on Red Hat Enterprise Linux (RHEL). This article describes how to install .NET on RHEL.
```
Comment (omajid) [inline docs/core/install/linux-rhel.md:11]:
```suggestion
.NET is supported on Red Hat Enterprise Linux (RHEL). This article describes how to install .NET on RHEL and CentOS Stream.
```

**Comment by omajid** [comment]
Thanks for the PR. I have some questions, but it's a great improvement even if those aren't addressed :+1:

**Review by BillWagner — APPROVED** [review]
This LGTM @adegeo 

Once you address the remaining comments, you can :shipit:

File: docs/core/install/linux-rhel.md [hunk 3]
```diff
@@ -173,6 +175,18 @@ source scl_source enable rh-dotnet31
 
 As an alternative to the ASP.NET Core Runtime, you can install the .NET Core Runtime that doesn't include ASP.NET Core support: replace `rh-dotnet31-aspnetcore-runtime-3.1` in the commands above with `rh-dotnet31-dotnet-runtime-3.1`.
 
+## CentOS Stream 9 ✔️
+
+.NET is included in the AppStream repositories for CentOS Stream 9. However, .NET Core 3.1 and .NET 5 have been removed from CentOS Stream 9 and you should use .NET 6. For more information, see the blog post [Using .NET with OpenSSL in CentOS Stream 9 | Omair Majid](https://omairmajid.com/posts/2021-08-25-using-.net-in-centos-stream-9/).
```
[the remark this answers is not in the document]
Comment (adegeo) [reply docs/core/install/linux-rhel.md:180]: @omajid I thought Stream 9 was already out? I generally don't have any preview versions in the docs. 😿

[the remark this answers is not in the document]
Comment (adegeo) [reply docs/core/install/linux-rhel.md:183]: I think it's OK to leave here though, the point of stream is it's just continually updated right? and you can use the install commands that the article documents, now. Correct?

**Pull request merged**